File: 3683-earliest-time-to-finish-one-task/3683-earliest-time-to-finish-one-task.cs
public class Solution {
    public int EarliestTime(int[][] tasks) {
       List<int> times=new List<int>();
       for(int i=0;i<tasks.Length;i++){
         int time=tasks[i].Aggregate((a,b)=>a+b);
         times.Add(time);
       }
        times.Sort((a,b)=>a-b);
        return times[0];
    }
}

File: 3683-earliest-time-to-finish-one-task/3683-earliest-time-to-finish-one-task.ts
function earliestTime(tasks: number[][]): number {
    let times=[];
    for(let i=0;i<tasks.length;i++){
        let time=tasks[i].reduce((a,b)=>a+b);
        times.push(time)
    }
    let result=times.sort((a,b)=>a-b);
    return result[0];
};